From df8027fb4f56b3d9bb17d26347f65f63669d99d2 Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Sat, 16 Oct 2021 22:57:15 +0300
Subject: [PATCH 120/233] [FIO toup] fit_image: use sha256 algo for images hash
 signing

The crc32 hash is weak. Use strong sha256 for signing images.

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 tools/fit_image.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/fit_image.c b/tools/fit_image.c
index 15f7c82d619..517d5cda4ea 100644
--- a/tools/fit_image.c
+++ b/tools/fit_image.c
@@ -203,7 +203,7 @@ static void get_basename(char *str, int size, const char *fname)
 static void add_crc_node(void *fdt)
 {
 	fdt_begin_node(fdt, "hash-1");
-	fdt_property_string(fdt, FIT_ALGO_PROP, "crc32");
+	fdt_property_string(fdt, FIT_ALGO_PROP, "sha256");
 	fdt_end_node(fdt);
 }
 
-- 
2.40.1

